src/graph.h

```diff
@@ -0,0 +1,60 @@
+/*	A Graph is an ordered set of vertices and edges.
+	Both, the Adjacency List and Adjacency Matrix implementation of
+	a graph are supported.
+
+	Usage :
+		Define a graph with 'struct Graph *'
+
+	Operations :
+		`INIT_GRAPH()`: Initiate an empty graph
+		`ADD_EDGE()`: Adds an edge between two nodes with a weight
+		`TO_LIST()`: List implementation of the graph
+		`TO_MATRIX()`: Matrix implementation of the graph
+
+	Example :
+*/
```

Comment:
For descriptive comments, see [L1-L29](https://github.com/OrkoHunter/nxcpy/blob/master/src/stack.h#L1-L29).